### PR TITLE
Update bloop to 1.4.6-33-1c6f6712

### PR DESCRIPTION
**Pull request by lolgab**
- Remove Scala Native deprecations

### CODE REVIEW

**Review by lefou — COMMENTED** [review]
Looks good to me.

**Comment by lolgab** [comment]
@lefou Rebased on top of current master.

File: contrib/bloop/test/src/mill/contrib/bloop/BloopTests.scala [hunk 1]
```diff
@@ -25,7 +25,7 @@ object BloopTests extends TestSuite {
 
     object scalaModule extends scalalib.ScalaModule with testBloop.Module {
       def scalaVersion = "2.12.8"
-      val bloopVersion = "1.4.6"
+      val bloopVersion = "1.4.6-33-1c6f6712"
```
Comment (lefou) [inline contrib/bloop/test/src/mill/contrib/bloop/BloopTests.scala:28]: If the intention is, to use the same version for testing as for building, then we should probably think about generating some `Versions` file in `build.sc` and use it here, as we do in other modules.

Comment (lolgab) [reply contrib/bloop/test/src/mill/contrib/bloop/BloopTests.scala:28]: ✅  Done!

Comment (lefou) [reply contrib/bloop/test/src/mill/contrib/bloop/BloopTests.scala]: Nice. I'm fine to merge after CI succeedes. Your PRs are a pleasure.

**Pull request merged**